xrootdfs/fs.py

```diff
@@ -145,7 +146,7 @@ def _stat_flags(self, path):
         status, stat = self.client.stat(self._p(path))
 
         if not status.ok:
```

Comment:
I think we should just check against ``status.ok``. I assume that ``status.error`` should like be considered together with ``status.fatal`` to determine how bad the error is, but we should ask Elvin instead.